myria3d/metrics/iou.py

```diff
@@ -0,0 +1,17 @@
+from torch import Tensor
+
+
+def iou(confmat: Tensor):
```

Comment:
why do you redefine iou?

Reply:
From MulticlassJaccardIndex we use this function `iou` to compute class-by-class IoUs. The previous method consisted of having k JaccardIndex metric object, one by class, with an inelegant modification that enabled to log single-class IoUs. 
That's actually a refactor that is not totally related to the torch upgrade :d

Reply:
ok, then th'as the reason why a lot of logging stuff disappeared?
then maybe worth adding to the changelog

Reply:
Will do !